src/components/Recommendation.jsx

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import Card from "./VideoCard";
+
+const Container = styled.div`
+  flex: 2;
+`;
+
+const Recommendation = ({ tags }) => {
+    const [ videos, setVideos ] = useState([]);
+
+    useEffect(() => {
+        const fetchVideos = async () => {
+            const res = await axios.get(`/videos/tag?tags=${tags}`);
+            setVideos(res.data.data);
+        };
+        fetchVideos();
+    }, [ tags ]);
+
+    return (
+        <Container>
+            {videos.map((video) => (
+                <Card type="sm" key={video._id} video={video} />
+            ))}
+        </Container>
+    );
+};
+
+export default Recommendation;
```

Comment:
every code file should have a line break at the end of the file. Can refer here: https://eslint.org/docs/latest/rules/eol-last